server/scripts/export.js

```diff
@@ -26,55 +33,52 @@ const roles = [
+        columns.set('Operator Contact Email', result.body.operatorEmail);
+        columns.set('Operator Contact Phone', result.body.operatorPhone);
+        columns.set('Business Operator Address', result.body.operatorAddress);
+        columns.set('Operator Postal Code', result.body.operatorPostalCode);
+        columns.set('Site Name', result.body.siteName);
+        columns.set('Site Address', result.body.address);
+        columns.set('Health Authority', result.body.healthAuthority);
+        columns.set('Site Contact First Name', result.body.siteContactFirstName);
+        columns.set('Site Contact Last Name', result.body.siteContactLastName);
+        columns.set('Site Contact Email', result.body.emailAddress);
+        columns.set('Site Contact Phone Number', result.body.phoneNumber);
+        columns.set('Site Type', result.body.siteType);
+        columns.set('Other Site Type', result.body.otherSite);
+        columns.set('Number of Public Long Term Care Beds', result.body.numPublicLongTermCare);
+        columns.set('Number of Private Long Term Care Beds', result.body.numPrivateLongTermCare);
+        columns.set('Number Public Assisted Living Beds', result.body.numPublicAssistedLiving);
 
-        roles.forEach(role => {
-          fields = {
-            ...fields,
-            [`${role} Current Casual`]: result.body.workforceBaseline.find(item => item.role === role).currentCasual,
-            [`${role} Current Full Time`]: result.body.workforceBaseline.find(item => item.role === role).currentFullTime,
-            [`${role} Vacancy Casual`]: result.body.workforceBaseline.find(item => item.role === role).vacancyCasual,
-            [`${role} Current Part Time`]: result.body.workforceBaseline.find(item => item.role === role).currentPartTime,
-            [`${role} Vacancy Full Time`]: result.body.workforceBaseline.find(item => item.role === role).vacancyFullTime,
-            [`${role} Vacancy Part Time`]: result.body.workforceBaseline.find(item => item.role === role).vacancyPartTime,
-          };
+        roles.forEach((role) => {
+          columns.set(`${role} Current Full Time`, result.body.workforceBaseline.find((i) => i.role === role).currentFullTime);
+          columns.set(`${role} Current Part Time`, result.body.workforceBaseline.find((i) => i.role === role).currentPartTime);
+          columns.set(`${role} Current Casual`, result.body.workforceBaseline.find((i) => i.role === role).currentCasual);
+          columns.set(`${role} Vacancy Full Time`, result.body.workforceBaseline.find((i) => i.role === role).vacancyFullTime);
+          columns.set(`${role} Vacancy Part Time`, result.body.workforceBaseline.find((i) => i.role === role).vacancyPartTime);
+          columns.set(`${role} Vacancy Casual`, result.body.workforceBaseline.find((i) => i.role === role).vacancyCasual);
         });
 
-        csvStream.write(fields);
+        columns.set('Number of Private Assisted Living Beds', result.body.numPrivateAssistedLiving);
```

Comment:
This should be line 67 I think, before the roles fields

Reply:
This change was made late last night at Pam's request. Wasn't something that the ticket captured.